site/sass/components/_fifty-fifty.scss

```diff
@@ -0,0 +1,61 @@
+@use 'sass:math';
+
+.fifty-fifty {
+  $single-column: 653px;
+  $content-width-m: pxRem(415px);
+  $content-width-r: generateScaling(($single-column 1440px), (340px 490px));
+
+  display: grid;
+  gap: 2rem 1.25rem;
+  grid-template-columns: repeat(auto-fit, minmax(pxRem(280px), 1fr));
+  justify-items: center;
+
+  &__content {
+    max-width: $content-width-m;
+    text-align: center;
+
+    @supports (margin-left: clamp(1px, 1vw, 2px)) {
+      max-width: $content-width-r;
+    }
+
+    @include mq($single-column) {
+      text-align: start;
+
+      @supports not (text-align: start) {
+        [dir='ltr'] & {
+          text-align: left;
+        }
+
+        [dir='rtl'] & {
+          text-align: right;
+        }
+      }
```

Comment:
```suggestion
```

While I appreciate the use of `@supports` here, the browsers that don't support `text-align: start` also don't support `@supports`, and the delta is ~1% globally and all super old browsers that we don't have primary support for, so I'm OK leaving this out.